default-port.patch

```diff
@@ -0,0 +1,12 @@
+diff -ura package.orig/config.example.yml package.new/config.example.yml
```

Comment:
Maybe, we should edit the defaults for the directories as well. Like in the bin aur package https://aur.archlinux.org/cgit/aur.git/tree/gotify-arch-defaults.patch?h=gotify-server-bin

Reply:
Ok, so everything stays as it is then. I'll edit.